components/navigation/navigation.css

```diff
@@ -36,6 +36,61 @@
   background-color: var(--status-dnd);
 }
 
+/* Drawer */
+
+.drawer {
+  display: flex;
+  justify-content: center;
+  border: 1px solid var(--text-color);
+  border-radius: var(--border-radius);
+}
+
+.drawer main,
+.drawer aside {
+  max-height: 400px;
```

Comment:
you can use relative units. but its ok to use. you can change it later

Reply:
Okay, thank you.